lib/MusicBrainz/Server/Report/EventsWithEAANoTypes.pm

```diff
@@ -0,0 +1,40 @@
+package MusicBrainz::Server::Report::EventsWithEAANoTypes;
+use Moose;
+
+with 'MusicBrainz::Server::Report::EventReport',
+     'MusicBrainz::Server::Report::FilterForEditor::EventID';
+
+sub component_name { 'EventsWithEaaNoTypes' }
+sub query {
+    '
+        SELECT
+            e.id AS event_id,
+            row_number() OVER (ORDER BY e.name COLLATE musicbrainz)
+        FROM (
+            SELECT DISTINCT e.*
+            FROM event e
+            WHERE EXISTS (
+                SELECT 1 FROM event_art_archive.event_art WHERE event_art.event = e.id
+            )
+            AND NOT EXISTS (
+                SELECT 1 FROM event_art_archive.event_art_type JOIN event_art_archive.event_art ON event_art_type.id = event_art.id WHERE event_art.event = e.id
```

Comment:
would be nice to break up this overly-long line a bit, you can also use `<<~'SQL'` for the query if you feel like it
```suggestion
                SELECT 1 FROM event_art_archive.event_art_type
                JOIN event_art_archive.event_art ON event_art_type.id = event_art.id
                WHERE event_art.event = e.id
```

Reply:
Moved it to use `<<~'SQL'`